src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -67,10 +71,11 @@ export class TelemetryComponent extends CdForm implements OnInit {
         this.moduleEnabled = configResp['enabled'];
         this.sendToUrl = configResp['url'];
         this.sendToDeviceUrl = configResp['device_url'];
+        this.showContactInfo = configResp['channel_ident'];
         this.options = _.pick(resp[0], this.requiredFields);
-        const configs = _.pick(configResp, this.requiredFields);
+        this.configResp = _.pick(configResp, this.requiredFields);
```

Comment:
@nizamial09 What minor front-end tweaks are you referring to? The `telemetry status`  response is similar to `this.mgrModuleService.getConfig('telemetry')` which is saved in `this.configResp`.  Personally I think, for this fix exposing new API was not required as we have all the data needed. 

```
{
channel_basic: true
channel_crash: true
channel_device: true
channel_ident: false
channel_perf: false
contact: ""
description: ""
device_url: "https://telemetry.ceph.com/device"
enabled: false
interval: 24
last_opt_revision: 1
leaderboard: false
log_level: ""
log_to_cluster: false
log_to_cluster_level: "info"
log_to_file: false
organization: ""
proxy: ""
url: "https://telemetry.ceph.com/report"
}
```

Reply:
Alright, in that case please ignore my comment!